### PR TITLE
Bump release 4.5.0

### PR DESCRIPTION
**Pull request by vcerenu**
This PR bump 4.5.0 version into master

### CODE REVIEW

File: multi-node/config/wazuh_cluster/wazuh_worker.conf [hunk 1]
```diff
@@ -222,8 +222,8 @@
   <global>
     <white_list>127.0.0.1</white_list>
     <white_list>^localhost.localdomain$</white_list>
-    <white_list>4.4.0.1</white_list>
-    <white_list>4.4.0.2</white_list>
+    <white_list>4.5.0.1</white_list>
+    <white_list>4.5.0.2</white_list>
```
Comment (alberpilot) [inline multi-node/config/wazuh_cluster/wazuh_worker.conf:226]: Why?

File: single-node/config/wazuh_cluster/wazuh_manager.conf [hunk 2]
```diff
@@ -222,8 +222,8 @@
   <global>
     <white_list>127.0.0.1</white_list>
     <white_list>^localhost.localdomain$</white_list>
-    <white_list>4.4.0.1</white_list>
-    <white_list>4.4.0.2</white_list>
+    <white_list>4.5.0.1</white_list>
+    <white_list>4.5.0.2</white_list>
```
Comment (alberpilot) [inline single-node/config/wazuh_cluster/wazuh_manager.conf:226]: Why?

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the question

**Review by teddytpc1 — APPROVED** [review]
LGTM

File: multi-node/config/wazuh_cluster/wazuh_worker.conf (hunk 1 above)
Comment (vcerenu) [reply multi-node/config/wazuh_cluster/wazuh_worker.conf]: It was an automatic modification, it was rolled back

**Review by alberpilot — APPROVED** [review]

**Pull request merged**